src/components/user/user.service.ts

```diff
@@ -363,13 +362,10 @@ export class UserService {
     // Update email
     if (input.email) {
       // Remove old emails and relations
-      const uniqueProperties: UniqueProperties<User> = {
-        email: ['Property', 'EmailAddress'],
-      };
       await this.db
         .query()
         .match([node('node', ['User', 'BaseNode'], { id: user.id })])
-        .call(setPropLabelsAndValuesDeleted, uniqueProperties)
+        .call(deleteProperties(User, 'email'))
```

Comment:
How does this work since 'email' is not an array?

Reply:
It's defined as variable args `(...args: string[])`. You pass them in one by one and JS gives them to the function as an array.